docs/02-StewardForValidators.md

```diff
@@ -0,0 +1,75 @@
+# Validators Instructions for Setting Up Steward
+
+Steward is an application intended for developers and validators on the Sommelier network.
+
+It can run as a server for forearding cellar contract calls from Strategy Providers through the Sommelier chain, or it can run in test mode to directly interact with Ethereum contracts as a single signer.
+
+It integrates the full functionality of gorc for operating as an orchestrator and relayer of [Gravity bridge](https://github.com/PeggyJV/gravity-bridge/) messages between the Ethereum and Cosmos chains.
+
+Stewards works in conjunction with the Orchestrator, so both processes must be running to fully participate in Cellar management.
+
+## Quickstart
+
+To start Steward, simply run
+
+```bash
+steward -c [path to your config toml] start
+```
+
+## Setting Up Steward
+
+In this section, let’s explore setting up steward for validators. There are two ways that validators will use Steward:
+
+1. Running Steward as a gRPC server for relaying SP contract calls through the chain. If a validator is not running this server, it cannot participate in Cellar management.
```

Comment:
This is the first time in this doc that "SP" is referenced, we should define it.